### PR TITLE
Feature/template functions

### PR DESCRIPTION
**Pull request by domingobishop**

### CODE REVIEW

File: page-section-landing.php [hunk 1]
```diff
@@ -8,16 +8,58 @@
...
-                        get_template_part( 'partials/feature-box' );
-                    endwhile; ?>
+			<?php while ( have_posts() ) : the_post(); ?>
+				<div class="row">
+					<div class="col-md-12">
+						<?php
+						global $post;
+						$image           = make_path_relative_no_pre_path( get_feature_image_url( $post->ID, 'feature-box-thumb' ) );
+						$title           = get_the_title();
+						$content         = get_the_content();
```
[the remark this answers is not in the document]
Comment (domingobishop) [reply page-section-landing.php]: wpautop added

File: page-section-landing.php [hunk 2]
```diff
@@ -8,16 +8,58 @@
...
+						'exclude'     => array( $post->ID ),
+						'child_of'    => $id,
+						'parent'      => $id,
+					) );
+					foreach ( $pages as $page ) {
+						$redirect               = get_post_meta( $page->ID, 'redirectUrl', true );
+						$boxes[ $i ]['title']   = $page->post_title;
+						$boxes[ $i ]['url']     = ( $redirect ) ? $redirect : make_path_relative( get_page_link( $page->ID ) );
+						$boxes[ $i ]['image']   = make_path_relative_no_pre_path( get_feature_image_url( $page->ID, 'landing-page-children-thumb' ) );
+						$boxes[ $i ]['excerpt'] = ( $page->post_excerpt ) ? $page->post_excerpt : first_sentence( $page->post_content );
```
[the remark this answers is not in the document]
Comment (domingobishop) [reply page-section-landing.php]: <p> wrap added

File: inc/functions/template-parts.php [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+function get_page_banner( $type, $title, $image='', $content='', $button='', $url='', $section_content='' ) {
+	if ( $type == 'level one' ) { ?>
+		<article class="banner feature-img feature-img-bg" <?php echo $image; ?>>
+			<div class="entry-header">
+				<h1><?php echo $title; ?></h1>
+			</div>
+			<?php if ( $content ) { ?>
+				<div class="entry-content">
+					<div class="col-xs-9 page-content">
+						<?php echo $content; ?>
```
[the remark this answers is not in the document]
Comment (domingobishop) [reply inc/functions/template-parts.php]: wpautop added

File: inc/functions/template-parts.php [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+							<h2>
+								<?php if ( !empty( $box['url'] ) ) { ?>
+									<a href="<?php echo $box['url'] ?>">
+										<?php echo $box['title'] ?>
+									</a>
+								<?php } else { echo $box['title']; } ?>
+							</h2>
+						</div>
+						<div class="entry-content clearfix">
+							<?php echo $box['content'] ?>
```
[the remark this answers is not in the document]
Comment (domingobishop) [reply inc/functions/template-parts.php]: wpautop added

**Comment by punalsc** [comment]
All ok.

**Pull request merged**